### PR TITLE
Vanilla DLC Weapons

### PR DESCRIPTION
**Pull request by ante185**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Adds conditional checks for adding marksmen DLC rifles and MGs to thew vanilla templates.
Minor adjustments and tweaks to the regular loadouts, such as colour matching using dlc attachments which are not restricted.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

Each tampered template has been loaded with all DLC enabled - no issues reported in RPT or by the template validator.

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Enable DLCs, load templates, observe enemy machine gunners, snipers, and marksmen.

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_SDK.sqf [hunk 1]
```diff
@@ -60,15 +60,11 @@ private _initialRebelEquipment = [
 "B_FieldPack_blk","B_FieldPack_cbr","B_FieldPack_green_F","B_FieldPack_khk","B_FieldPack_oli",
 "V_Chestrig_blk","V_Chestrig_rgr","V_Chestrig_khk","V_Chestrig_oli","V_BandollierB_blk","V_BandollierB_rgr",
 "V_BandollierB_oli","V_Rangemaster_belt","V_TacChestrig_cbr_F","V_TacChestrig_oli_F","V_TacChestrig_grn_F",
-"Binocular",
+"Binocular","hgun_Pistol_Signal_F","6Rnd_GreenSignal_F","6Rnd_RedSignal_F",
 "acc_flashlight","acc_flashlight_smg_01","acc_flashlight_pistol"
 ];
+_initialRebelEquipment append [["launch_RPG7_F", 15], ["RPG7_F", 45]];
```
Comment (Bob-Murphy) [inline A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_SDK.sqf:66]: Did you lose the (allowDLCExpansion) condition?

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_SDK.sqf (hunk 1 above)
Comment (ante185) [reply A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_SDK.sqf:66]: SDK forces the APEX expansion, the if statement would have always been true

Comment (ante185) [reply A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_SDK.sqf:66]: I can change it back, do what we do in the FIA template, but then I'd wanna remove the force forceDLC[] = {"expansion"} from the SDK entry in template.hpp as adding that condition back is not gonna change anything as is.

Comment (jaj22) [reply A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_SDK.sqf:66]: Nah, this is fine. Maybe make a note at the top of templates about what DLC they require, but yeah, this one is Apex-specific.

Comment (Bob-Murphy) [reply A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Reb_SDK.sqf:66]: Gotcha

**Review by Bob-Murphy — APPROVED** [review]

**Pull request merged**